src/tic-tac-toe.ts

```diff
@@ -0,0 +1,24 @@
+type Square = string;
+type Row = Square[];
+type Board = Row[];
+
+class Game {
+  public board: Board;
+  private currentPlayer: string;
+  constructor() {
+    this.currentPlayer = "X";
+    this.board = [["", "", ""], ["", "", ""], ["", "", ""]];
```

Comment:
It would be interesting to see this generalized to an NxN board!

Reply:
Added ability to dictate board size when a new game is created. Defaults to a 3x3 board.